practices/simpleDesign/readme.md

```diff
@@ -0,0 +1,35 @@
+# Simple Design
+
+The simplest design is often (almost always) the best solution to a given problem. The simplest solution is the easiest to maintain and the easiest to change (assuming you have appropriate testing of course). The assertion here is that trying to "build for the future" or "build in extensibility" ends up being more costly as we cannot predict what future requirements will be. Solve for the now, iterate on future later.
```

Comment:
"build for the future" or "build in extensibility" are both common and valid requests, so I'm a little hesitant include this wording. (build in extensibility is essentially advocating following SOLID principles, and if we're not building for the future we're kind of shooting ourselves in the foot.) A simple and elegant solution often provides extensibility and future reuse _without adding complexity_. The added complexity of "making sure that if we switch from a messaging based async platform to exclusively synchronous service calls our consumers won't be impacted" (not bitter) is what should be avoided.

Reply:
This is actually the current description on the front page for core values, but I can change it here and there!